### PR TITLE
Release/23.01

### PR DESCRIPTION
**Pull request by jwoo-msft**
Porting packages updates to main

1. Corrected designer version to match what's in npmjs
2. Incremented designer version
3. updated adaptivecard version
4. corrected control version to match what's in npmjs.
5. updated dependencies after audit run.

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://api.github.com/repos/microsoft/AdaptiveCards/pulls/8280)

### CODE REVIEW

File: source/nodejs/adaptivecards-designer/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "adaptivecards-designer",
-	"version": "2.5.0-alpha.0",
+	"version": "2.4.3",
```
Comment (anna-dingler) [inline source/nodejs/adaptivecards-designer/package.json:3]: Why did we change this?

Comment (jwoo-msft) [reply source/nodejs/adaptivecards-designer/package.json:3]: @anna-dingler this is the value used in npm js

Comment (anna-dingler) [reply source/nodejs/adaptivecards-designer/package.json:3]: I thought we switched to the alpha annotation for lerna bootstrap?

Comment (jwoo-msft) [reply source/nodejs/adaptivecards-designer/package.json:3]: all right, thanks, i will fix it.

**Pull request merged**